test/views/returns/credits/CreditsClaimedListViewSpec.scala

```diff
@@ -35,24 +32,35 @@ class CreditsClaimedListViewSpec extends ViewSpecBase with ViewAssertions{
   private val page = inject[CreditsClaimedListView]
   private val form = new CreditsClaimedListFormProvider()()
 
-  private val summaryRows = Seq(
-    SummaryListRow(
-      key = Key(Text("exported")),
-      value = Value(Text("answer")),
-      actions = Some(Actions(items = Seq(
+  private val rows = Seq(
+    CreditSummaryRow(
+      label = "exported",
+      value = "answer",
+      actions = Seq(
```

Comment:
these staying as sequence?

Reply:
Yes, I changed because I use the CreditSummaryRow to display the  Credit total row too, which has no actions. So there where few way to do that, including using an Option, but the Seq seem easer to handle it. Open to any suggestions.